benchmark/compare/compare_travis.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+echo $TRAVIS_GO_VERSION
+echo $TRAVIS_COMMIT_RANGE
+IFS='...' read -r -a commits <<< "$TRAVIS_COMMIT_RANGE"
+echo "base commit number:"
+echo ${commits[0]}
+echo "current commit number:"
+echo ${commits[-1]}
+
+if [ -d "benchmark/compare" ]; then
+  echo "dir benchmark/compare exist"
+  go get -d -v -t google.golang.org/grpc/...
+
+  cp -r benchmark tmpbenchmark
+
+  go test google.golang.org/grpc/benchmark/... -benchmem -bench=BenchmarkClient/-Tracing-kbps_0-MTU_0-maxConcurrentCalls_1-latency_0s- | tee result1
+  ls benchmark/compare/
```

Comment:
What's the purpose of this?
Also, the output on travis from this is `ls: cannot access benchmark/compare/: No such file or directory`